### DISCUSSION TITLE
Error during installation -  running cloudformation

### DISCUSSION THREAD

**Discussion by tomg-formpak** [awslabs/aws-saas-boost#282] [Q&A]
**Uploading 27 Lambda functions to S3
Running CloudFormation
Awaiting CloudFormation Stack sb-dev to complete.  Sleep 5 minute(s)...
Awaiting CloudFormation Stack sb-dev to complete.  Sleep 1 minute(s)...
Awaiting CloudFormation Stack sb-dev to complete.  Sleep 1 minute(s)...
Awaiting CloudFormation Stack sb-dev to complete.  Sleep 1 minute(s)...
Awaiting CloudFormation Stack sb-dev to complete.  Sleep 1 minute(s)...
CloudFormation Stack: sb-dev failed.
===========================================================
Installation Error: Error with CloudFormation stack sb-dev. Check the events in the AWS CloudFormation Console
Please see the detailed log file saas-boost-install.log

On checking, there are differences in the s3 bucket for the old (v1.1.2) and the new version (v2.0.0-rc1).In v1.1.2,  a folder named "lambdas/" is created in the S3 bucket whereas in the new version there is no lambda folder but the files are named "lambdas\filenamexxxx.zip".  Due to this installation is getting failed.**

**Comment by PoeppingT** [comment]
Hi @tomg-formpak , can you clarify why you think this is your failing installation? What version of SaaS Boost are you trying to install, and do you already have existing SaaS Boost installations in your account?

**Reply by tomg-formpak** [reply]
I have already tried v1.1.2 and it was successful on installation. Now I went with  v2.0.0-rc1 since it has more features that suit our business. On installation  v2.0.0-rc1, its getting failed on creating cloudformation stack (please see on the question)

I think it's caused by the issue  I mentioned above _"a folder named "lambdas/" is created in the S3 bucket whereas in the new version there is no lambda folder but the files are named "lambdas\filenamexxxx.zip"_  because in the YAML file it's searching for a source folder "lambda" in S3 bucket, that folder is not present in the S3 bucket instead it's present as a file ( as mentioned above). But this was there in the v1.1.2 when I compared. I checked this with an AWS consultant also. 

I guess this is the reason, not sure if this is the main reason or not. I will be retrying the installation again for the v2.0.0-rc1 on a  clean aws account today, I will then update with a screenshot or log.

**Reply by brtrvn** [reply]
Hi Tom, I would suggest you install from `HEAD` of the `main` branch not the v2.0.0-rc1 release. We have made many bug fixes to v2 since we posted that release candidate and you'll be better off using the latest for your testing until we can finalize a v2 release.

To understand what the errors are that you are seeing, you need to look at the installation log and/or the CloudFormation console for the failed stacks to see what resources failed to provision.
